runtime/src/main/java/com/dylibso/chicory/runtime/GlobalInstance.java

```diff
@@ -15,13 +15,23 @@ public GlobalInstance(Value value) {
     }
 
     public GlobalInstance(Value value, MutabilityType mutabilityType) {
-        this.value = value.raw();
+        this.values = new long[] {value.raw()};
```

Comment:
so, even though I don't want to block, I am tempted to suggest using a pair of long fields instead of an array; this way we avoid some pointer chasing. Even if apparently we "waste" more memory, it's really about the same or less, because the array costs a reference + the array itself (header+n*sizeof(longs)), while in this case we only have 2 longs.

Reply:
agree, I'll update the PR soon :+1: thanks!

Reply:
done :+1: